docs/streams/upgrade-guide.html

```diff
@@ -44,6 +49,27 @@ <h1>Upgrade Guide &amp; API Changes</h1>
         See <a href="#streams_api_changes_0101">below</a> a complete list of 0.10.1 API changes that allow you to advance your application and/or simplify your code base, including the usage of new features.
     </p>
 
+    <h3><a id="streams_api_changes_110" href="#streams_api_changes_110">Streams API changes in 1.1.0</a></h3>
+
+    <p>
+        The introduction of <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-220%3A+Add+AdminClient+into+Kafka+Streams%27+ClientSupplier">KIP-220</a>
+        enables you to provide configuration parameters for the embedded admin client created by Kafka Streams, similar to the embedded producer and consumer clients.
+        You can provide the configs via the <code>StreamsConfig</code> by adding the configs with the prefix as defined by <code>StreamsConfig#adminClientPrefix(String)</code>
+        to distinguish them from configurations of other clients that share the same config names.
+    </p>
+
+    <p>
+        New methods in <code>KafkaStreams</code>:
+    </p>
+    <ul>
+        <li> added overload for the constructor that allows to override the <code>Time</code> object used for tracking system wall-clock time; this is useful for unit testing your application code. </li>
+    </ul>
+
+    <p> New methods in <code>KafkaClientSupplier</code>: </p>
+    <ul>
+        <li> added <code>#getAdminClient(config)</code> that allows to overwrite an <code>AdminClient</code> used for administrative requests such as internal topic creations, etc. </li>
```

Comment:
Do we need the `#` prefix? Also, do you mean "override"?

Reply:
Good point, I think we do not.